ruby3.2-ffi-compiler.yaml

```diff
@@ -54,3 +54,5 @@ update:
   github:
     identifier: ffi/ffi-compiler
     use-tag: true
+  ignore-regex-patterns:
+    - "2.0.0"
```

Comment:
The regex expressions should not be surrounded by double quotes. An example can be seen here: https://github.com/chainguard-dev/enterprise-packages/blob/03548c804c11133bf2c42a9ddeee79395f8fd193/kyverno-policy-reporter-fips.yaml#L57

Reply:
I appreciate the example! I was going based on https://github.com/chainguard-dev/melange/blob/main/docs/UPDATE.md#ignore-versions and wasn't 100% sure.

Reply:
Looks to be resolved